src/model/documents/category.ts

```diff
@@ -0,0 +1,8 @@
+import { BaseModel } from '../base-model';
+import { CategoryType } from '../enums/category-type';
+
+export interface Category extends BaseModel {
+  imagePath: string;
+  name: string;
+  type: CategoryType;
```

Comment:
Why there is a "type"? IMO, It should be a reference to another, existing category (if it is nested) or by default it should be "main screen".

Reply:
I don't understand. Did you refer to the documentation?
`type` is predefined string that should be stored as enum.
![image](https://user-images.githubusercontent.com/48626088/113477398-c3b45700-9481-11eb-9ad3-e7e1cb6dbc35.png)

Reply:
Why should category have it's own type? 
You can create Category and category reflects the group in the terminal.
Products, especially meals/recipes & articles should have category, but also the category can be nested in another category, which should be reflected in terminal. By default, new category should be placed in the "main screen" on terminal. I know that Poster has option to add category to ingredients also, but it serves different purpose and is less important IMO.
That's why I was asking about your implementation of "type" property.

Reply:
We discussed the topic on Slack and agreed that I would remove the type and add a parent category to the model. Done.